CHANGELOG.md

```diff
@@ -3,14 +3,47 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [[1.0.0](https://github.com/sanger-tol/blobtoolkit/releases/tag/1.0.0)] – ??? - [2023-04-06]
 
-Initial release of nf-core/blobtoolkit, created with the [nf-core](https://nf-co.re/) template.
+### Enhancements & fixes
 
-### `Added`
+Initial release of sanger-tol/blobtoolkit :tada
```

Comment:
```suggestion
Initial release of sanger-tol/blobtoolkit :tada:
```

Reply:
You should also list somewhere the major differences with the Snakemake version, so that people don't expect a drop-in replacement, incl.
- Reads have to be aligned separately
- Two sub-workflows are still in progress
- The yaml config file is not used to control the actual pipeline
- The yaml config file does not reflect the actual software versions used by the pipeline, and may have other discrepancies